### PR TITLE
Web dev agent tools improvements

### PR DESCRIPTION
**Pull request by mitya52**

### CODE REVIEW

File: src/tools/tools_description.rs [hunk 1]
```diff
@@ -310,12 +320,19 @@ tools:
...
           reload
+      - name: "tab"
+        type: "string"
+        description: "Tab where command will be executed."
+      - name: "screenshot"
+        type: "boolean"
+        description: "Optionally make a screenshot of a tab."
     parameters_required:
       - "command"
+      - "tab"
```
Comment (olegklimov) [inline src/tools/tools_description.rs:335]: Parameters are unclear.
1. Two ways to make a screenshot: "screenshot" command as in examples, and "screenshot" boolean parameter.
2. Tab is a string, a required parameter? How does it know what string to write there?

File: src/integrations/integr_chrome.rs [hunk 2]
```diff
@@ -150,84 +159,93 @@ async fn start_chrome_session(
...
 
 async fn interact_with_chrome(
+    tab_name: &String,
     command_args: &Vec<String>,
+    append_screenshot: &bool,
     session_hashmap_key: &String,
     tool_call_id: &String,
     gcx: Arc<ARwLock<GlobalContext>>) -> Result<Vec<ContextEnum>, String>
 {
+    let mut force_screenshot = append_screenshot.clone();
```
Comment (olegklimov) [inline src/integrations/integr_chrome.rs:186]: you don't have to clone bool

File: src/integrations/integr_chrome.rs [hunk 3]
```diff
@@ -150,84 +159,93 @@ async fn start_chrome_session(
...
             .map_err(|e| e.to_string())?;
-        let url = chrome_session.tab.get_url();
+        let url = tab.get_url();
         let response = client.get(url.clone()).send().await.map_err(|e| e.to_string())?;
         if !response.status().is_success() {
-            content = format!("Unable to fetch url: {}; status: {}", url, response.status());
+            tool_log.push(format!("Unable to fetch url: {}; status: {}", url, response.status()));
         } else {
-            content = response.text().await.map_err(|e| e.to_string())?;
+            tool_log.push(response.text().await.map_err(|e| e.to_string())?);
```
Comment (olegklimov) [inline src/integrations/integr_chrome.rs:232]: I'd recommend putting more simple facts about the web page into tool logs. 

Sent https://finance.yahoo.com/ to tab 1
Completed operation in 0.4s
Current tabs:

[
{"tab_index": 0, "url": "https://www.google.com/", "mobile": true},
{"tab_index": 1, "url": "https://finance.yahoo.com/"},
{"tab_index": 2, "url": "http://EpicFails.com"},
]

File: src/tools/tools_description.rs (hunk 1 above)
Comment (mitya52) [reply src/tools/tools_description.rs:335]: screenshot command needed to monitor current state of the tab without reloading it

**Pull request closed without merging**